frontend/src/app/projects/project-detail/project-users/project-user-settings.component.ts

```diff
@@ -50,7 +50,7 @@ export class ProjectUserSettingsComponent implements OnChanges {
     this.permissionRequiredValidator()
   );
   constructor(
-    public repoUserService: RepositoryUserService,
+    public projectUserService: RepositoryUserService,
```

Comment:
We should also revise the `RepositoryUserService` and change the naming from `Repository` to `Project` or?

Reply:
Done.